### PR TITLE
Update DIN token faucet support

### PR DESCRIPTION
**Pull request by wangminqi**
Update DIN token faucet support

### Description

add a description of your changes here...

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

File: cmd/faucet/customized/README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ We will review the request, and once it is approved, the faucet tool will start
 
 # 2.Token List
 ## 2.1.DemoToken
```
Comment (zzzckck) [inline cmd/faucet/customized/README.md:11]: pls create a new item: ##2.2. DIN

File: cmd/faucet/customized/README.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ We will review the request, and once it is approved, the faucet tool will start
...
-- symbol: DEMO
-- amount: 10000000000000000000
-- icon: ./demotoken.png
-- addr: https://testnet.bscscan.com/address/0xe15c158d768c306dae87b96430a94f884333e55d
-- fundTx: [0xa499dc9aaf918aff0507538a8aa80a88d0af6ca15054e6acc57b69c651945280](https://testnet.bscscan.com/tx/0x2a3f334b6ca756b64331bdec9e6cf3207ac50a4839fda6379e909de4d9a194ca)
+- symbol: DIN
+- amount: 9999000000000000000000
```
Comment (zzzckck) [inline cmd/faucet/customized/README.md:13]: this is the amount that user will get for each faucet request.

Comment (wangminqi) [reply cmd/faucet/customized/README.md:13]: I see I will adjust

File: cmd/faucet/customized/README.md (hunk 1 above)
Comment (wangminqi) [reply cmd/faucet/customized/README.md]: added

File: cmd/faucet/customized/README.md (hunk 2 above)
Comment (wangminqi) [reply cmd/faucet/customized/README.md]: adjusted. 10 DIN

**Review by zzzckck — APPROVED** [review]

**Review by Kutasy — APPROVED** [review]

**Review by zlacfzy — APPROVED** [review]

**Pull request merged**